GSA/digital-gov-drupal
# Digital-32: Migrate top level pages

## Jira ticket




[DIGITAL-32](https://cm-jira.usa.gov/browse/DIGITAL-32)

## Purpose



This Pr is setting up and and starting the migration of the top level landing pages and setting up the node teasers for the content types

This is taking the creation of views and others to setup the content that is rendered on the landing pages. 

Lading pages are all pages that are in the menu such as Blogs, Events, Guides, etc

Follow up tasks that have been ticketed out:
1. fix/update logos for short posts 
2. address and fix how the list on pages like events and news are rendering in regards to the order 
3. fixing limitation with colons in drupal classes for the guides page 
4. setting up job board -> this is going to take more discussions on how this will be set up 
5. setting up resouces pages -> this will need us to set up another option for topics on how they will be grouped on the page 
6. menu on blog page is not active 

things to check for reviewer: 
1. check the work for communities, the cards are not wrapping how they are on the live site not sure if i didn't set up the view right or not. 
2. review the todos in the code and check to see if there is something i missed that can be done now

## Includes the following PRs that must be merged first



## Deployment and testing

### Local Setup



Pull down branch and run `lando retune -y`

### QA/Testing instructions


1. Visit pages and check overall style and set up and compare to live site
3. review template pages that are commented as "Template that was updated", after running the twig lint there were files pushed up that were not a part of the pr but fixed for code standards

## Checklist for the Developer




## Checklist for the Peer Reviewers
